src/python/pants/backend/docker/util_rules/docker_build_context_test.py

```diff
@@ -41,13 +43,20 @@ def rule_runner() -> RuleRunner:
             *package_pex_binary.rules(),
             *parser_rules(),
             *pex_from_targets.rules(),
+            *shell_target_types_rules(),
```

Comment:
Glad you figured that out that you need this for the target generation rule to work. Hopefully that wasn't too tricky!

Reply:
Not tricky at all. Think I added it right away.. ;)